.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.21"
+          go-version: "1.23"
```

Comment:
[FYI] The workflow files in celestia-app use

```yaml
        with:
          go-version-file: 'go.mod'
```

so that we can update the go version in the root go.mod and then all the workflows use that version.

https://github.com/celestiaorg/celestia-app/blob/31b5d01b06dbfcd637ff25c2e3deaf4d678f140f/.github/workflows/lint.yml#L12-L14

Reply:
Yeah I prefer that too. Will update to use `go-version-file`

Reply:
I updated in https://github.com/celestiaorg/cosmos-sdk/pull/431/commits/5192f00b174c6031efcb1a0da99babf33eda783b if you want to try trigger workflows again and merge if all g 🙏🏻

Reply:
Triggered workflows. Also added @damiannolan @julienrbrt as writers on this repo so 🤞 that gives you permission to trigger workflows if you need to

Reply:
ty @rootulp!